Jerrod/Assignments/TDD/beginning_test/projectClass.rb

```diff
@@ -0,0 +1,22 @@
+class Project
+  attr_accessor :name, :description, :team
+  def initialize(name, desc)
+    @name = name
+    @description = desc
+    @team = []
+    self
```

Comment:
Initialize doesn't need self, it's technically called by new so it's not what is returned :)